addon/mixins/touch-action.js

```diff
@@ -41,7 +41,7 @@ export default Mixin.create({
   ignoreTouchAction: false,
 
   init() {
-    this._super(...arguments);
```

Comment:
I'm still not sure why we don't want this. ember recommends we do this in all hooks, so I think we should leave it.

Reply:
Updated.

FWIW it is a possible perf hit simply because of the babel transpilation. I tried measuring the differences in a dummy app with 200+ components to see if it made a difference and the results were pretty much the same.